generative_ai/inference/stream_text_basic.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def generate_content(PROJECT_ID: str, REGION: str, MODEL_ID: str) -> object:
+    # [START generativeaionvertexai_stream_text_basic]
+    import vertexai
+
+    from vertexai.generative_models import GenerativeModel
+
+    vertexai.init(project=PROJECT_ID, location=REGION)
+
+    model = GenerativeModel(MODEL_ID)
+    responses = model.generate_content(
+        "Write a story about a magic backpack.", stream=True
+    )
+
+    for response in responses:
+        print(response)
```

Comment:
Comment that this loop can take a while. Basically using `stream=True` moves the waiting from the `generate_content(...)` call to this loop.

Reply:
It's a good point. Because these samples work together with the API documentation, I will see if we can add some messages directly in the API to that effect so we don't need to repeat the message in each sample. Thanks.